invenio_communities/permissions.py

```diff
@@ -162,6 +162,8 @@ class CommunityPermissionPolicy(BasePermissionPolicy):
     # correct permissions based on which the field will be exposed only to moderators
     can_moderate = [Disable()]
 
+    can_manage_communities = [UserManager, SystemProcess()]
```

Comment:
Possible name clash with `can_manage_access`, maybe `can_administrate_communities` would be better? maybe we can come up together with something more suitable. otherwise LGTM!